src/integration-test/java/com/commercetools/project/sync/InventorySyncMultiChannelIT.java

```diff
@@ -1,206 +1,249 @@
-//                    .supplyChannel(supplyChannelReference)
-//                    .build()))
-//        .toCompletableFuture();
-//  }
-//
-//  @Test
-//  void run_asInventoryFullSync_ShouldNotReturnAnError() {
-//    // test
-//    CliRunner.of().run(new String[] {"-s", "inventoryEntries", "-f"}, createITSyncerFactory());
-//
-//    // assertions
-//    assertThat(testLogger.getAllLoggingEvents())
-//        .allMatch(loggingEvent -> !Level.ERROR.equals(loggingEvent.getLevel()));
-//
-//    // Every sync module is expected to have 2 logs (start and stats summary)
-//    assertThat(testLogger.getAllLoggingEvents()).hasSize(2);
-//  }
-// }
+package com.commercetools.project.sync;
+
+import static com.commercetools.project.sync.util.CtpClientUtils.CTP_SOURCE_CLIENT;
+import static com.commercetools.project.sync.util.CtpClientUtils.CTP_TARGET_CLIENT;
+import static com.commercetools.project.sync.util.IntegrationTestUtils.cleanUpProjects;
+import static com.commercetools.project.sync.util.IntegrationTestUtils.createITSyncerFactory;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.commercetools.api.client.ProjectApiRoot;
+import com.commercetools.api.models.channel.Channel;
+import com.commercetools.api.models.channel.ChannelDraft;
+import com.commercetools.api.models.channel.ChannelDraftBuilder;
+import com.commercetools.api.models.channel.ChannelRoleEnum;
+import com.commercetools.api.models.common.LocalizedString;
+import com.commercetools.api.models.inventory.InventoryEntry;
+import com.commercetools.api.models.inventory.InventoryEntryDraft;
+import com.commercetools.api.models.inventory.InventoryEntryDraftBuilder;
+import com.commercetools.api.models.type.CustomFieldsDraftBuilder;
+import com.commercetools.api.models.type.FieldContainer;
+import com.commercetools.api.models.type.FieldContainerBuilder;
+import com.commercetools.api.models.type.FieldDefinition;
+import com.commercetools.api.models.type.FieldDefinitionBuilder;
+import com.commercetools.api.models.type.FieldTypeBuilder;
+import com.commercetools.api.models.type.ResourceTypeId;
+import com.commercetools.api.models.type.TypeDraft;
+import com.commercetools.api.models.type.TypeDraftBuilder;
+import com.commercetools.project.sync.inventoryentry.InventoryEntrySyncer;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import io.vrap.rmf.base.client.ApiHttpResponse;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.IntStream;
+import javax.annotation.Nonnull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import uk.org.lidalia.slf4jext.Level;
+import uk.org.lidalia.slf4jtest.TestLogger;
+import uk.org.lidalia.slf4jtest.TestLoggerFactory;
+
+@Disabled(
+    "Disabled for normal builds, "
+        + "This scenario created to evaluate multiple channel use case"
+        + "see: https://github.com/commercetools/commercetools-project-sync/issues/301")
```

Comment:
According to the creator the test should stay disabled: https://github.com/commercetools/commercetools-project-sync/pull/310 (`Then I disabled that test because it creates lots of resources and running it for every build might be problematic and unnecessary`).

I'm also not sure if we should keep it or delete it forever. It's not useful as disabled, but on the other hand thanks to this test we know there was this problem and in case the same problem occurs again in the future, we could use it.

Reply:
Since I cannot decide, should we keep the test here as it was before? 😅

Reply:
Let's keep it.